scipy/stats/_qmc.py

```diff
@@ -914,13 +910,41 @@ class LatinHypercube(QMCEngine):
 
         .. versionadded:: 1.8.0
 
+    orthogonal : bool, optional
+        Orthogonal array based LHS of strength 2 [7]_, [8]_.
+        When used, only ``n=p**2`` points can be sampled, with ``p`` a prime
+        number. It also constrains ``d <= p + 1``.
+        Default is False.
+
+        .. versionadded:: 1.8.0
+
     seed : {None, int, `numpy.random.Generator`}, optional
         If `seed` is None the `numpy.random.Generator` singleton is used.
         If `seed` is an int, a new ``Generator`` instance is used,
         seeded with `seed`.
         If `seed` is already a ``Generator`` instance then that instance is
         used.
 
+    Notes
+    -----
+
+    When LHS is used for integrating a function :math:`f` over :math:`n`,
+    LHS is extremely effective on integrands that are nearly additive [2]_.
+    With a LHS of :math:`n` points, the variance of the integral is always
+    lower than plain MC on :math:`n-1` points [3]_. There is a central limit
+    theorem for LHS on the mean and variance of the integral [4]_, but not
+    necessarily for optimized LHS due to the randomization.
```

Comment:
I think this is much clearer, thank you! I might suggest a few tweaks later now that i understand, but already i think it is more approachable. Thanks!